dev-next-release - v1.13.0

xrutayisire opened the pull request:
Pull Request for next release

xrutayisire committed:
ci(alpha): Make alpha deployment run directly

xrutayisire committed:
Publish

 - cimsirp@1.12.1-dev-next-release.0
 - sveltekit@1.12.1-dev-next-release.0
 - @slicemachine/adapter-next@0.3.15-dev-next-release.0
 - @slicemachine/adapter-nuxt@0.3.15-dev-next-release.0
 - @slicemachine/adapter-nuxt2@0.3.15-dev-next-release.0
 - @slicemachine/adapter-sveltekit@0.3.15-dev-next-release.0
 - @slicemachine/init@2.6.2-dev-next-release.0
 - @slicemachine/manager@0.11.1-dev-next-release.0
 - @slicemachine/plugin-kit@0.4.15-dev-next-release.0
 - slice-machine-ui@1.12.1-dev-next-release.0
 - start-slicemachine@0.11.5-dev-next-release.0

xrutayisire committed:
feat(sliceZone): Remove slice from slice zone (#1130)

xrutayisire committed:
feat(sliceZone): Only Add dropdown in Slice Zone header (#1131)

xrutayisire committed:
Publish

 - cimsirp@1.12.1-dev-next-release.1
 - sveltekit@1.12.1-dev-next-release.1
 - @slicemachine/adapter-next@0.3.15-dev-next-release.1
 - @slicemachine/adapter-nuxt@0.3.15-dev-next-release.1
 - @slicemachine/adapter-nuxt2@0.3.15-dev-next-release.1
 - @slicemachine/adapter-sveltekit@0.3.15-dev-next-release.1
 - @slicemachine/init@2.6.2-dev-next-release.1
 - @slicemachine/manager@0.11.1-dev-next-release.1
 - @slicemachine/plugin-kit@0.4.15-dev-next-release.1
 - slice-machine-ui@1.12.1-dev-next-release.1
 - start-slicemachine@0.11.5-dev-next-release.1

hypervillain committed:
Next slice templates - AlternateGrid (#1132)

hypervillain committed:
Publish

 - cimsirp@1.12.1-dev-next-release.2
 - sveltekit@1.12.1-dev-next-release.2
 - @slicemachine/adapter-next@0.3.15-dev-next-release.2
 - @slicemachine/adapter-nuxt@0.3.15-dev-next-release.2
 - @slicemachine/adapter-nuxt2@0.3.15-dev-next-release.2
 - @slicemachine/adapter-sveltekit@0.3.15-dev-next-release.2
 - @slicemachine/init@2.6.2-dev-next-release.2
 - @slicemachine/manager@0.11.1-dev-next-release.2
 - @slicemachine/plugin-kit@0.4.15-dev-next-release.2
 - slice-machine-ui@1.12.1-dev-next-release.2
 - start-slicemachine@0.11.5-dev-next-release.2

xrutayisire committed:
feat(Page Type): Specific blank slate right after Page Type creation (#1133)

xrutayisire committed:
Publish

 - cimsirp@1.12.1-dev-next-release.3
 - sveltekit@1.12.1-dev-next-release.3
 - @slicemachine/adapter-next@0.3.15-dev-next-release.3
 - @slicemachine/adapter-nuxt@0.3.15-dev-next-release.3
 - @slicemachine/adapter-nuxt2@0.3.15-dev-next-release.3
 - @slicemachine/adapter-sveltekit@0.3.15-dev-next-release.3
 - @slicemachine/init@2.6.2-dev-next-release.3
 - @slicemachine/manager@0.11.1-dev-next-release.3
 - @slicemachine/plugin-kit@0.4.15-dev-next-release.3
 - slice-machine-ui@1.12.1-dev-next-release.3
 - start-slicemachine@0.11.5-dev-next-release.3

hypervillain committed:
Next template ctlogos (#1134)

hypervillain committed:
Publish

 - cimsirp@1.12.1-dev-next-release.4
 - sveltekit@1.12.1-dev-next-release.4
 - @slicemachine/adapter-next@0.3.15-dev-next-release.4
 - @slicemachine/adapter-nuxt@0.3.15-dev-next-release.4
 - @slicemachine/adapter-nuxt2@0.3.15-dev-next-release.4
 - @slicemachine/adapter-sveltekit@0.3.15-dev-next-release.4
 - @slicemachine/init@2.6.2-dev-next-release.4
 - @slicemachine/manager@0.11.1-dev-next-release.4
 - @slicemachine/plugin-kit@0.4.15-dev-next-release.4
 - slice-machine-ui@1.12.1-dev-next-release.4
 - start-slicemachine@0.11.5-dev-next-release.4

xrutayisire committed:
feat(SliceTemplates): Remove unnecessary import

xrutayisire committed:
Publish

 - cimsirp@1.12.1-dev-next-release.5
 - sveltekit@1.12.1-dev-next-release.5
 - @slicemachine/adapter-next@0.3.15-dev-next-release.5
 - @slicemachine/adapter-nuxt@0.3.15-dev-next-release.5
 - @slicemachine/adapter-nuxt2@0.3.15-dev-next-release.5
 - @slicemachine/adapter-sveltekit@0.3.15-dev-next-release.5
 - @slicemachine/init@2.6.2-dev-next-release.5
 - @slicemachine/manager@0.11.1-dev-next-release.5
 - @slicemachine/plugin-kit@0.4.15-dev-next-release.5
 - slice-machine-ui@1.12.1-dev-next-release.5
 - start-slicemachine@0.11.5-dev-next-release.5

xrutayisire committed:
feat(SliceTemplates): Update copy

xrutayisire committed:
Publish

 - cimsirp@1.12.1-dev-next-release.6
 - sveltekit@1.12.1-dev-next-release.6
 - @slicemachine/adapter-next@0.3.15-dev-next-release.6
 - @slicemachine/adapter-nuxt@0.3.15-dev-next-release.6
 - @slicemachine/adapter-nuxt2@0.3.15-dev-next-release.6
 - @slicemachine/adapter-sveltekit@0.3.15-dev-next-release.6
 - @slicemachine/init@2.6.2-dev-next-release.6
 - @slicemachine/manager@0.11.1-dev-next-release.6
 - @slicemachine/plugin-kit@0.4.15-dev-next-release.6
 - slice-machine-ui@1.12.1-dev-next-release.6
 - start-slicemachine@0.11.5-dev-next-release.6

gijsbotje commented on packages/slice-machine/lib/builders/CustomTypeBuilder/SliceZone/index.tsx:
```diff
@@ -223,7 +223,7 @@ const SliceZone: React.FC<SliceZoneProps> = ({
                         startIcon={<Icon name="contentCopy" />}
                       >
-                        Slice templates
+                        From templates
```
Shouldn't this also be singular?
```suggestion
                        From template
```

xrutayisire replied:
Thank you for your proposal 🙂

But for this one, we want to focus on the fact you can add from multiple templates at once. 
On the other hand, we choose "From library" because most of our users only have one library and when you have multiple libraries of slices you will understand. It's to not confuse new people. Anyway, we are working on a better wording for "library" 🤞

xrutayisire committed:
fix(ci): Fix slice machine unit tests

xrutayisire committed:
Publish

 - cimsirp@1.12.1-dev-next-release.7
 - sveltekit@1.12.1-dev-next-release.7
 - @slicemachine/adapter-next@0.3.15-dev-next-release.7
 - @slicemachine/adapter-nuxt@0.3.15-dev-next-release.7
 - @slicemachine/adapter-nuxt2@0.3.15-dev-next-release.7
 - @slicemachine/adapter-sveltekit@0.3.15-dev-next-release.7
 - @slicemachine/init@2.6.2-dev-next-release.7
 - @slicemachine/manager@0.11.1-dev-next-release.7
 - @slicemachine/plugin-kit@0.4.15-dev-next-release.7
 - slice-machine-ui@1.12.1-dev-next-release.7
 - start-slicemachine@0.11.5-dev-next-release.7

xrutayisire committed:
fix(sm.json): Remove old ref to sm.json (#1136)

xrutayisire committed:
Publish

 - cimsirp@1.12.1-dev-next-release.8
 - sveltekit@1.12.1-dev-next-release.8
 - @slicemachine/adapter-next@0.3.15-dev-next-release.8
 - @slicemachine/adapter-nuxt@0.3.15-dev-next-release.8
 - @slicemachine/adapter-nuxt2@0.3.15-dev-next-release.8
 - @slicemachine/adapter-sveltekit@0.3.15-dev-next-release.8
 - @slicemachine/init@2.6.2-dev-next-release.8
 - @slicemachine/manager@0.11.1-dev-next-release.8
 - @slicemachine/plugin-kit@0.4.15-dev-next-release.8
 - slice-machine-ui@1.12.1-dev-next-release.8
 - start-slicemachine@0.11.5-dev-next-release.8

hypervillain committed:
feat(Slice Templates): Hero Slice Template 

Co-authored-by: Xavier Rutayisire <xavier.rutayisire@gmail.com>

xrutayisire committed:
Publish

 - cimsirp@1.12.1-dev-next-release.9
 - sveltekit@1.12.1-dev-next-release.9
 - @slicemachine/adapter-next@0.3.15-dev-next-release.9
 - @slicemachine/adapter-nuxt@0.3.15-dev-next-release.9
 - @slicemachine/adapter-nuxt2@0.3.15-dev-next-release.9
 - @slicemachine/adapter-sveltekit@0.3.15-dev-next-release.9
 - @slicemachine/init@2.6.2-dev-next-release.9
 - @slicemachine/manager@0.11.1-dev-next-release.9
 - @slicemachine/plugin-kit@0.4.15-dev-next-release.9
 - slice-machine-ui@1.12.1-dev-next-release.9
 - start-slicemachine@0.11.5-dev-next-release.9

xrutayisire committed:
feat(Slice Templates): Fix UI for slice templates

xrutayisire committed:
Publish

 - cimsirp@1.12.1-dev-next-release.10
 - sveltekit@1.12.1-dev-next-release.10
 - @slicemachine/adapter-next@0.3.15-dev-next-release.10
 - @slicemachine/adapter-nuxt@0.3.15-dev-next-release.10
 - @slicemachine/adapter-nuxt2@0.3.15-dev-next-release.10
 - @slicemachine/adapter-sveltekit@0.3.15-dev-next-release.10
 - @slicemachine/init@2.6.2-dev-next-release.10
 - @slicemachine/manager@0.11.1-dev-next-release.10
 - @slicemachine/plugin-kit@0.4.15-dev-next-release.10
 - slice-machine-ui@1.12.1-dev-next-release.10
 - start-slicemachine@0.11.5-dev-next-release.10

xrutayisire committed:
feat(Slice Templates): Fix missing image size in css

xrutayisire committed:
Publish

 - cimsirp@1.12.1-dev-next-release.11
 - sveltekit@1.12.1-dev-next-release.11
 - @slicemachine/adapter-next@0.3.15-dev-next-release.11
 - @slicemachine/adapter-nuxt@0.3.15-dev-next-release.11
 - @slicemachine/adapter-nuxt2@0.3.15-dev-next-release.11
 - @slicemachine/adapter-sveltekit@0.3.15-dev-next-release.11
 - @slicemachine/init@2.6.2-dev-next-release.11
 - @slicemachine/manager@0.11.1-dev-next-release.11
 - @slicemachine/plugin-kit@0.4.15-dev-next-release.11
 - slice-machine-ui@1.12.1-dev-next-release.11
 - start-slicemachine@0.11.5-dev-next-release.11
